alifd__next/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "alifd__next",
+  "filename": "next.min.js",
+  "version": "1.19.24",
```

Comment:
```suggestion
```
Please remove the version attr, the auto-update bot will add this itself once it pulls in versioned files.